src/app/legacy/containers/CpsRecommendations/RecommendationsPromo/index.jsx

```diff
@@ -100,12 +101,14 @@ const StyledHeadline = styled.div`
   align-items: center;
 `;
 
-const RecommendationsPromo = ({ promo, eventTrackingData }) => {
+const RecommendationsPromo = ({ promo, eventTrackingData, isArticle }) => {
```

Comment:
You can use the RequestContext to get the `pageType` to save passing down `isArticle`, like:

`const { pageType } = useContext(RequestContext);`

There is a variable called `ARTICLE_PAGE` that you can then string match to create `isArticle` e.g. https://github.com/bbc/simorgh/blob/9997384bcad94b0175a7b8123b373788492f0183/src/app/legacy/psammead/psammead-social-embed/src/CaptionWrapper/index.jsx#L43-L44